provers/evm-prover/ARCHITECTURE.md

```diff
@@ -0,0 +1,76 @@
+# EVM Prover
+
+The EVM Prover implements the Prover service for EVM rollups, creating zero-knowledge range proofs for state membership and state transitions. It uses the following SP1 programs to generate these range proofs.
```

Comment:
It doesn't do state membership. Is a range proof synonymous with an aggregate proof?


```suggestion
The EVM Prover implements the Prover service for EVM rollups, creating zero-knowledge range proofs for state transitions. It uses the following SP1 programs to generate these range proofs.
```

Reply:
Done.